pillar.example

```diff
@@ -66,3 +66,13 @@ postgresql:
       method: trust
       options: NOSUPERUSER NOCREATEDB NOCREATEROLE INHERIT LOGIN REPLICATION
       password:
+
+  databases:
+    - database: triebwerk
+      username: deploy
+      gisdb: true
```

Comment:
I don't like the `gisdb` keyword here. I think a generic field `extensions` would be better, and would work with all sorts of extensions (especially built-ins)

```yaml
- database: triebwerk
  extensions: [postgis]
```

Reply:
Yes, like the schema definition. I will change this accordingly.